compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -576,6 +577,8 @@ static LogicalResult
 setVectorDistributionConfig(IREE::GPU::TargetAttr target,
                             mlir::FunctionOpInterface entryPoint,
                             Operation *computeOp) {
+  if (!isROCmBackend(target))
+    return failure();
```

Comment:
This should have a comment. Why fail on just rocm? Is there some bug? Is something just not implemented yet?
(Comment should have enough info for someone in the future to determine when this `return failure()` is safe to remove)

Reply:
Good point. This pass is not yet supported for CUDA. Added comment to make it clear.

Reply:
Ah, thanks. I missed the `!` character too :P